src/v3/convertToRum.ts

```diff
@@ -122,6 +122,45 @@ export function getSpanSummaryAttributes<AllPossibleScopesT>(
   return spanAttributes
 }
 
+// type RoundableObject<T> = T extends object
```

Comment:
thought i needed this type but i guess i didnt. keeping it here in case we do!

if needed, will delete before merge